pyplanet/apps/contrib/brawl_match/__init__.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, *args, **kwargs):
 			await self.register_match_task(self.start_match, player)
 
 	async def check_maps_on_server(self):
-		return all(map(lambda t: self.instance.map_manager.playlist_has_map(t[0]), self.brawl_maps))
+		return all([self.instance.map_manager.playlist_has_map(map) for (map, timeout) in self.brawl_maps])
```

Comment:
Not really sure whats best here. 
`return all([self.instance.map_manager.playlist_has_map(map[0]) for map in self.brawl_maps])`
`return all([self.instance.map_manager.playlist_has_map(map) for (map, _) in self.brawl_maps])`
or to just leave it as is. They all have their advantages I guess. I myself prefer my first one over the other two, but you can decide for yourself if you want to keep it or change to one of the two i suggested.